report: March 2025 (#25)
apache/cordova-apache-board-reports: 2025/03.md

**Issues for the board:**

There are no issues requiring board attention at this time.

## Membership Data

There are currently 99 committers and 96 PMC members in this project.

The Committer-to-PMC ratio is roughly 1:1.

**Community changes, past quarter:**

- No new PMC members. Last addition was Pieter Van Poyer on 2021-04-06.
- No new committers. Last addition was Pieter Van Poyer on 2021-04-06.

## Project Activity

Our project made releases this quarter for our core platforms to keep up with current requirements.

**Releases:**

- cordova-plugin-media-capture@6.0.0 was released on 2025-02-26.
- cordova-common@5.0.1 was released on 2025-02-02.
- cordova-plugin-file@8.1.3 was released on 2024-11-20.

## Community Health

Overall, the community health is strong. The ASF Project Statistics gives the project a Community Health Score (Chi): 4.70 (Healthy)

We continue to see contributions from a dedicated group of individuals. The project remains stable and continues to see good traffic.

This quarter, we released a few patch updates and a major release. The major release focused on one of our plugins, ensuring compliance with Android's new permission requirements.

> Other contributors continue to focus on improvements and preparations for future major releases of Cordova-iOS and Cordova-Android. Some have gone above and beyond by reaching out to Android to raise awareness of issues and discuss solutions. This proactive approach provided clear direction, allowing us to begin preparing necessary changes to address these issues.

Reviewer: ```suggestion
A couple contributors continue to focus on improvements and preparations for future major releases of Cordova-iOS and Cordova-Android. Some have gone above and beyond by reaching out to Android to raise awareness of issues and discuss solutions. This proactive approach provided clear direction, allowing us to begin preparing necessary changes to address these issues.
```
Other sounds a little odd.

> On January 31st, Cordova hosted a virtual meetup to wrap up 2024 and kick off the new year. The main goal was to highlight Cordova's accomplishments and share what we hope for in 2025. We also gathered feedback from community members to understand their experiences with Cordova, how they use it, any questions they had, features they would like to see, or issues that needed investigation.

Reviewer: ```suggestion
On January 31st, Cordova hosted a virtual meetup to wrap up 2024 and kick off the new year. The main goal was to highlight Cordova's accomplishments and share what we hope to work on in 2025. We also gathered feedback from community members to understand their experiences with Cordova, how they use it, any questions they had, features they would like to see, or issues that needed investigation.
```

> This was the first meetup we've hosted in a long time, and we hope to continue these events for information sharing and community engagement. A total of 117 people registered, with around 70 have attended.

Reviewer: ```suggestion
This was the first meetup we've hosted in a long time, and we hope to continue these events for information sharing and community engagement. A total of 117 people registered, with around 70 have attended. Among the meetups attendees we have seen a sentiment that Cordova is stable for many projects and users rely on it for their apps. We also saw some interest in contributing and maintaining from new faces, but they don't know where to start. For this reason we are discussing and working on some ideas to make contributing, testing and releasing Cordova easier.
```

I added few words about my impression from the meetup and our current ideas. Add this if you like it.
